docs/cody/clients/install-cli.mdx

```diff
@@ -0,0 +1,212 @@
+
+First, make sure you have installed the necessary tools to use `cody auth login`.
+
+<Tabs>
+  <Tab title="macOS">
+The pre-installed `security` tool is used. No additional installation is required.
+  </Tab>
+  <Tab title="Windows">
+The [CredentialManager](https://www.powershellgallery.com/packages/CredentialManager/2.0) PowerShell module is required.
+Open a PowerShell terminal with administrator privileges and run the following command:
+
+```powershell
+Install-Module -Name CredentialManager
+```
+
+  </Tab>
+  <Tab title="Linux (Ubuntu)">
+The [`secret-tool`](https://manpages.ubuntu.com/manpages/focal/man1/secret-tool.1.html#:~:text=secret%2Dtool%20is%20a%20command,of%20attribute%20keys%20and%20values) command-line tool is used. Run the following commands to install it:
+```shell
+sudo apt install libsecret-tools
+sudo apt install gnome-keyring
+```
+  </Tab>
+</Tabs>
+
+Run the following command to authenticate the Cody CLI:
+
+<Tabs>
+  <Tab title="Browser (fastest)">
+```shell
+cody auth login --web
+```
+This will open a browser window where you can authenticate with your Sourcegraph account.
+Close the browser tab after authentication is complete.
+  </Tab>
+  <Tab title="Command Line">
+- For Cody Pro/Free accounts, create an access token at https://sourcegraph.com/user/settings/tokens.
+- For Cody Enterprise accounts, sign into your Sourcegraph Enterprise account and create an access token under `Account > Settings > Access Tokens`.
+```shell
+export SRC_ENDPOINT=ENDPOINT
+export SRC_ACCESS_TOKEN=ACCESS_TOKEN
+cody auth login # stores the access token securely
+```
+You don't need to export `SRC_ENDPOINT` or `SRC_ACCESS_TOKEN` after running `cody auth login`.
+Subsequent command invocation will use the stored access token.
+  </Tab>
+</Tabs>
+
+Confirm that the authentication was successful by running the following command:
+
+```shell
+cody auth whoami
+# ✔ Authenticated as USERNAME on ENDPOINT
+```
+
+## Authenticate with only environment variables
+
+Skip this step if you have already authenticated with the `cody auth login` command.
+
+If you prefer not to let Cody CLI store your access token, you can also pass the endpoint URL and access token through the environment variables `SRC_ENDPOINT` and `SRC_ACCESS_TOKEN`.
+
+<Tabs>
+  <Tab title="macOS/Linux">
+```shell
+export SRC_ENDPOINT=ENDPOINT
+export SRC_ACCESS_TOKEN=ACCESS_TOKEN
+```
+  </Tab>
+  <Tab title="Windows">
+```powershell
+$env:SRC_ENDPOINT = "ENDPOINT"
+$env:SRC_ACCESS_TOKEN = "ACCESS_TOKEN"
+```
+  </Tab>
+</Tabs>
+
+It's recommended to store these access tokens in a secure location. 
+For example, you can store them with a password manager like [1Password](https://1password.com/) or [Bitwarden](https://bitwarden.com/).
+
+It may be tempting to export these variables in your shell startup script, but this is not recommended because it will expose your access token to all commands you run from the terminal.
```

Comment:
```suggestion
It is not recommended to export these variables in your shell startup script because it will expose your access token to all commands you run from the terminal. Instead, consider sourcing these environment variables on-demand when you need to authenticate with the Cody CLI.
```

Reply:
Much clearer, thank you.